src/docfx/lib/HtmlUtility.cs

```diff
@@ -361,6 +442,50 @@ internal static HtmlNode StripTags(this HtmlNode html)
             return html;
         }
 
+        internal static List<Error> SecurityScan(this HtmlNode html, Document file)
+        {
+            var errors = new List<Error>();
+
+            foreach (var node in html.DescendantsAndSelf())
+            {
+                if (node.NodeType != HtmlNodeType.Element)
+                {
+                    continue;
+                }
+
+                // ignore herf. will implement in another feature. todo
+                if (node.Name == "a")
```

Comment:
why ignore `a`?

Reply:
href' schema check will be in another feature, here we should include `a` to check its attributs

---
In reply to: [414339124](https://github.com/dotnet/docfx/pull/5823#discussion_r414339124) [](ancestors = 414339124)